pkg/trace/info/info.go

```diff
@@ -327,7 +328,7 @@ func initInfo(conf *config.AgentConfig) error {
 			return fmt.Sprintf("%02.1f", v*100)
 		},
 	}
-	expvar.NewInt("pid").Set(int64(os.Getpid()))
+	expvar.NewString("pid").Set(strconv.Itoa(os.Getpid()))
```

Comment:
Does this conversion get done on every request to the info endpoint? Or just when it's initialized?

Reply:
Just once at initialization